File: Frontend/Frontend/src/app/demo/api/Models/MetodosPagoViewModel.ts
export class MetodosPagoViewModel {
    mtPag_Id!: number;
    mtPag_Descripcion!: string;
    mtPag_UsuarioCreacion!: number;
    mtPag_FechaCreacion!: Date;
    mtPag_UsuarioModificacion?: number | null;
    mtPag_Estado?: boolean | null;
    mtPag_FechaModificacion?: Date | null;
    usuarioCreacion?: string;
    usuarioModificacion?: string;
}

export class MetodoPagoEnviar {
    mtPag_Id:number;
    mtPag_Descripcion: string;
}

export class Fill {
    mtPag_Id!: number;
    mtPag_Descripcion!: string;
    usuarioCreacion?: string;
    usuarioModificacion?: string;
    mtPag_FechaCreacion!: Date;
    mtPag_FechaModificacion?: Date | null;
}

File: Backend/Sistema_Larach/Sistema_Larach.Common/Models/MetodosPagoViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sistema_Larach.Common.Models
{
   public class MetodosPagoViewModel
    {
        public int MtPag_Id { get; set; }
        public string MtPag_Descripcion { get; set; }
        public int MtPag_UsuarioCreacion { get; set; }
        public DateTime MtPag_FechaCreacion { get; set; }
        public int? MtPag_UsuarioModificacion { get; set; }
        public bool? MtPag_Estado { get; set; }
        public DateTime? MtPag_FechaModificacion { get; set; }
        [NotMapped]
        public string UsuarioCreacion { get; set; }
        [NotMapped]
        public string UsuarioModificacion { get; set; }
    }
}
